python/tank/authentication/shotgun_authenticator.py

```diff
@@ -36,13 +36,13 @@ class ShotgunAuthenticator(object):
     A simple use case scenario would look something like this::
 
         # create an authenticator
-        sa = ShotgunAuthenticator()
+        sg = ShotgunAuthenticator()
```

Comment:
`sa` was on purpose, for ShotgunAuthenticator.

When we have `sg` in code samples usually it's for shotgun connections